src/witan/gias.clj

```diff
@@ -573,156 +573,153 @@
+  ;;    | :reason-establishment-opened-code | ReasonEstablishmentOpened (code) |                         Reason establishment opened (code) |      :int16 |    51090 |          0 |      0.000 |      99.00 |
+  ;;    | :reason-establishment-opened-name | ReasonEstablishmentOpened (name) |                                Reason establishment opened |     :string |    50364 |        726 |            |            |
+  ;;    |                        :open-date |                         OpenDate |                                                  Open date | :local-date |    20789 |      30301 | 1800-01-01 | 2025-04-01 |
+  ;;    | :reason-establishment-closed-code | ReasonEstablishmentClosed (code) |                         Reason establishment closed (code) |      :int16 |    51090 |          0 |      0.000 |      99.00 |
+  ;;    | :reason-establishment-closed-name | ReasonEstablishmentClosed (name) |                                Reason establishment closed |     :string |    38016 |      13074 |            |            |
+  ;;    |                       :close-date |                        CloseDate |                                                 Close date | :local-date |    23633 |      27457 | 1900-01-01 | 2034-08-31 |
+  ;;    |          :phase-of-education-code |          PhaseOfEducation (code) |                                  Phase of education (code) |      :int16 |    51090 |          0 |      0.000 |      7.000 |
+  ;;    |          :phase-of-education-name |          PhaseOfEducation (name) |                                         Phase of education |     :string |    51090 |          0 |            |            |
+  ;;    |                :statutory-low-age |                  StatutoryLowAge |                                            Age range (low) |      :int16 |    47126 |       3964 |      0.000 |      19.00 |
+  ;;    |               :statutory-high-age |                 StatutoryHighAge |                                           Age range (high) |      :int16 |    47129 |       3961 |      3.000 |      99.00 |
+  ;;    |                    :boarders-code |                  Boarders (code) |                                            Boarders (code) |      :int16 |    51090 |          0 |      0.000 |      9.000 |
+  ;;    |                    :boarders-name |                  Boarders (name) |                                                   Boarders |     :string |    49406 |       1684 |            |            |
+  ;;    |           :nursery-provision-name |          NurseryProvision (name) |                                          Nursery provision |     :string |    51055 |         35 |            |            |
+  ;;    |         :official-sixth-form-code |         OfficialSixthForm (code) |                                 Official sixth form (code) |      :int16 |    51090 |          0 |      0.000 |      9.000 |
```

Comment:
Is this needed? I suspect you are including it to note the differences between the before and after updating the database but if you could confirm that @eraderna ?

Reply:
Yes, I've been keeping it as it helps detect changes in the edubaseall dataset from one update to the next.